contracts/IdentityOracle.sol

```diff
@@ -202,37 +160,27 @@ contract IdentityOracle is DAOUpgradeableContract, ChainlinkClient, Ownable {
         }
         return result;
     }
-
-    function setJobIDIPFS1(string memory _jobid) public {
-        _onlyAvatarTmp();
-        JOBID_GET_IPFS_CID1 = _jobid;
-    }
-
-    function setJobIDIPFS2(string memory _jobid) public {
-        _onlyAvatarTmp();
-        JOBID_GET_IPFS_CID2 = _jobid;
-    }
-
-    function setJobIDStateHash(string memory _jobid) public {
+    
+    function getJobIDStateHashIPFSCID() public view returns (string memory) {
         _onlyAvatarTmp();
-        JOBID_GET_STATE_HASH = _jobid;
+        return JOBID_GET_STATE_HASH_IPFSCID;
     }
 
-    function getJobIDIPFS1() public view returns (string memory) {
+    function setJobIDStateHashIPFSCID(string memory _jobid) public {
         _onlyAvatarTmp();
-        return JOBID_GET_IPFS_CID1;
+        JOBID_GET_STATE_HASH_IPFSCID = _jobid;
     }
-
-    function getJobIDIPFS2() public view returns (string memory) {
+    
+    function getCLNodeAdderss() public view returns (address) {
         _onlyAvatarTmp();
-        return JOBID_GET_IPFS_CID2;
+        return CHAINLINK_NODE_ADDRESS;
```

Comment:
no need for view method, make variable public

Reply:
fixed by https://github.com/GoodDollar/ChainlinkIdentityOracle/pull/5/commits/f05ce99726a869f1dfaf944456647138eaf25fa3